generators/app/templates/docker-compose.yml

```diff
@@ -5,8 +5,8 @@ services:
     image: <%= imageName %><% if (environment === 'debug') { %>:debug<% } %>
     build:
       context: .
-      dockerfile: Dockerfile<% if (environment === 'debug') { %>.debug<% } %><% if (isWebProject && portNumber) { %>
-    ports:
+      dockerfile: Dockerfile<% if (environment === 'debug') { %>.debug<% } %><% if ((isWebProject && portNumber) || (environment === 'debug' && debugPortNumber)) { %>
```

Comment:
>(environment === 'debug' && debugPortNumber) [](start = 116, length = 44)

Nitpick: this is a different way of doing the if than below. If both work please pick one and make them consistent (I favor this one as less verbose)